.github/workflows/app_check.yml

```diff
@@ -70,3 +70,28 @@ jobs:
     - name: PodLibLint FirebaseAppCheck Cron
       # TODO: Remove --allow-warnings when stabilized.
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseAppCheck.podspec --platforms=ios ${{ matrix.flags }}
+
+  spm:
+    # Don't run on private repo unless it is a PR.
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Initialize xcodebuild
+      run: scripts/setup_spm_tests.sh
+    - name: iOS Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh FirebaseAppCheck iOS spmbuildonly
```

Comment:
Any reason not to use `spm` instead of `spmbuildonly` to run the tests also?

Reply:
Ah I just worked off what I saw in the [`crashlytics.yml`](../blob/master/.github/workflows/crashlytics.yml).

How do we decide when to pass `spm` vs `spmbuildonly`?

Reply:
Oh, Im guessing it depends whether or not we have setup test targets for that library in the package manifest

Reply:
I believe it should be AppCheckUnit and AppCheckUnitSwift - similar to Database in #8456